core/server/api/mail.js

```diff
@@ -5,31 +5,25 @@ var _             = require('lodash').runInContext(),
     pipeline      = require('../utils/pipeline'),
     config        = require('../config'),
     errors        = require('../errors'),
-    GhostMail     = require('../mail'),
+    mail          = require('../mail'),
     Models        = require('../models'),
     utils         = require('./utils'),
     notifications = require('./notifications'),
     path          = require('path'),
-    fs            = require('fs'),
-    templatesDir  = path.resolve(__dirname, '..', 'mail', 'templates'),
-    htmlToText    = require('html-to-text'),
-    readFile      = Promise.promisify(fs.readFile),
     docName       = 'mail',
     i18n          = require('../i18n'),
     mode          = process.env.NODE_ENV,
     testing       = mode !== 'production' && mode !== 'development',
     mailer,
-    mail;
-
-_.templateSettings.interpolate = /{{([\s\S]+?)}}/g;
+    apiMail;
 
 /**
  * Send mail helper
  */
 
 function sendMail(object) {
-    if (!(mailer instanceof GhostMail) || testing) {
-        mailer = new GhostMail();
+    if (!(mailer instanceof mail.GhostMailer) || testing) {
+        mailer = new mail.GhostMailer();
```

Comment:
Looks like this PR has some mail refactoring in it - lets move that out to its own PR and get it merged first?